### PR TITLE
updated for 4..3.10; new issues 37650, 37660, 37680, and revised desc…

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
…ription of 36927.

### CODE REVIEW

File: docs/release_notes_128t_4.3.md [hunk 1]
```diff
@@ -2,10 +2,70 @@
...
+------
+- **I95-37577** LDAP authentication fails for users that contain a '-' in their name.
+------
+- **I95-37588** Value for `configure > authority > router > system > software-update > repository > address` uses the first lexigraphically sorted router for all other routers in authority instead of using a unique value per router.
+------
+- **I95-37650** Clickjacking vulnerability exists if the application allows content to be loaded inside a frame of an attacker-controlled webpage.  
+------
+- **I95-37660** Outdated python-pip package exposes vulnerability to sniffing, cross-origin redirect, or injection attacks.
+------
+- **I95-37680** nodeMonitor process may fault on shutdown.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.3.md:63]: `nodeMonitor process may fault on shutdown of 128T`

File: docs/release_notes_128t_4.3.md [hunk 2]
```diff
@@ -2,10 +2,70 @@
...
+------
+- **I95-37457** `show rib` and `show bgp` do not support more than one pagination session. 
+------
+- **I95-37513** Network interface cards that do not respond to physical stats may result in system lockup.
+------
+- **I95-37577** LDAP authentication fails for users that contain a '-' in their name.
+------
+- **I95-37588** Value for `configure > authority > router > system > software-update > repository > address` uses the first lexigraphically sorted router for all other routers in authority instead of using a unique value per router.
+------
+- **I95-37650** Clickjacking vulnerability exists if the application allows content to be loaded inside a frame of an attacker-controlled webpage.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.3.md:59]: `The 128T web UI incorrectly supports being embedded as an iFrame within another page`

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.3.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.3.md]: changed

File: docs/release_notes_128t_4.3.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.3.md]: changed

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — DISMISSED** [review]

**Review by jeffmart — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**